prime-router/metadata/fhir_mapping/hl7/datatypes/XON/XONExtension.yml

```diff
@@ -0,0 +1,32 @@
+url:
+  type: SYSTEM_URL
+  value: xon-organization
+
+extension:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - expressionType: nested
+      vars:
+        xon3: STRING_ALL, XON.3
```

Comment:
Is `STRING_ALL` needed here? Recently I've only been using that type for fields that repeat.

Reply:
I've been just not thinking about it 😅 and using it everywhere since it doesn't hurt to use it on a single value